README.md

```diff
@@ -20,7 +20,7 @@ You will need:
 Install dependencies:
 
 ```
-$ yarn install
+$ yarn install --production=false
```

Comment:
Oops I've just noticed that this comment thread is on the readme, not on travis.yml! I'll fix this 😆 😨